lnrpc/rpc.proto

```diff
@@ -1063,6 +1063,9 @@ message GetInfoResponse {
     /// Number of active channels
     uint32 num_active_channels = 4 [json_name = "num_active_channels"];
 
+    /// Number of inactive channels
+    uint32 num_inactive_channels = 15 [json_name = "num_inactive_channels"];
```

Comment:
IMO it's more important to keep the proto definitions in increasing order, so it stays clear what the last allocate number is. This also helps avoid gaps (where did 7 go?) and other inconsistencies.

In the past, we've reordered fields by defining a display struct at the RPC level with the desired order and mapping the fields into that before printing. There are a number of fields in getinfo that I think would benefit from a reordering, and this provides a better long-term solution by decoupling the display from the proto definitions

Reply:
Agreed! 👍